Tella/Domain/Entity/Report/ServerType.swift

```diff
@@ -21,8 +22,10 @@ extension ServerConnectionType {
             return LocalizableSettings.settServerReportsSuccessMessage.localized
         case .nextcloud:
             return LocalizableSettings.nextcloudSuccessMessage.localized
+        case.dropbox:
+            return LocalizableSettings.settServerDropboxSuccessMessage.localized
         default:
-            return ""
+            return LocalizableSettings.settServerDropboxSuccessMessage.localized
```

Comment:
maybe keeping empty message is better in default case ?

Reply:
oh yeah, this is a typo. Thanks for catching this